src/sass/shared/variables.scss

```diff
@@ -85,11 +91,21 @@ $font-size-title-xsmall: 21px;
 $font-size-small: 18px;
 $font-size-xsmall: 16px;
 $font-size-xxsmall: 14px;
+$font-size-micro: 12px;
 $font-size-footer-titles: 21px;
 $font-size-footer-text: $font-size-xsmall;
 $font-size-secondary-nav-item: 21px;
 $font-size-testimonial-usernames: 18px;
 
+// Line Height
+$line-height-condensed: 1em;
+$line-height-small: 1.2em;
+$line-height-projects-title: 30px;
+
+// Font weight
+$font-weight-bold: bold;
```

Comment:
I mean, a bold is a bold. It's not the same moving a size in pixels than moving something like bold. 

If now is bold you have:

```
font-weight: $font-weight-yada-yada;
```

instead of 

```
font-weight: bold;
```

But if in the future it's not bold, but normal, you won't change the value of `$font-weight-yada-yada` but drop it completely. That's why I think it makes no sense to make `bold` a variable. If we were dealing with multiple font weights in the design, maybe. But we only have regular or bold.